src/views/pages/events/eventdefault.js

```diff
@@ -1,16 +1,14 @@
-import {
-  Box, Button, Grid, makeStyles
-} from '@material-ui/core';
+import { Box, Button, Grid, makeStyles } from '@material-ui/core';
 import React, { useState, useEffect } from 'react';
 import BookmarkedEvents from 'src/components/BookmarkedEvents';
 import UserNewEvents from 'src/components/UserNewEvents';
 import UserUpcomingEvents from 'src/components/UserUpcomingEvents';
 import UserEventStats from 'src/components/UserEventStats';
-// import SearchBar from 'src/components/search';
+import SearchBar from 'src/components/search';
 import DrawerLayout from 'src/layouts/DrawerLayout';
 import { useHistory } from 'react-router-dom';
 
-const useStyles = makeStyles((theme) => ({
+const useStyles = makeStyles(theme => ({
```

Comment:
use func arrow-parens properly

Reply:
This all codes are not written by me that's why  I am facing more problem.